cypress/component/Auth/auth.spec.ts

```diff
@@ -1,29 +1,47 @@
 /* eslint-disable no-undef */
 
+import {OidcBroker} from '../../../src/libs/auth/oidcBroker';
 import {Auth} from '../../../src/libs/auth/auth';
-import {Config} from '../../../src/libs/config';
-import {GoogleIS} from '../../../src/libs/googleIS';
 import {OAuth2} from '../../../src/libs/ajax/OAuth2';
 import {Storage} from '../../../src/libs/storage';
-import { v4 as uuid } from 'uuid';
+import {v4 as uuid} from 'uuid';
+import {mockOidcUser} from './mockOidcUser';
 
-describe('Auth', function () {
+describe('Auth Failure', function () {
+  it('Sign In error throws expected message', async function () {
+    cy.stub(OidcBroker, 'signIn').returns(null);
+    cy.on('fail', (err) => {
+      return err.message !== Auth.signInError();
+    });
+    Auth.signIn();
+    expect(Storage.getOidcUser()).to.be.null;
+    expect(Storage.userIsLogged()).to.be.false;
+  });
+});
+
+describe('Auth Success', function () {
   // Intercept configuration calls
-  beforeEach(async () => {
+  beforeEach(() => {
     cy.intercept({
       method: 'GET',
       url: '/config.json',
       hostname: 'localhost',
     }, {'env': 'ci'});
     cy.stub(OAuth2, 'getConfig').returns({
-      'authorityEndpoint': 'authorityEndpoint',
+      'authorityEndpoint': Cypress.config().baseUrl,
       'clientId': 'clientId'
     });
-    await Auth.initialize();
+    Auth.initialize();
```

Comment:
This function still seems to be async, does it need to be `await`ed?

Reply:
I was initially confused by this. Cypress gives you a warning if you use `async` in `beforeEach`:
```
Cypress Warning: Cypress detected that you returned a promise in a test, but also invoked one or more cy commands inside of that promise.

The test title was:

  > Auth Success "before each" hook

While this works in practice, it's often indicative of an anti-pattern. You almost never need to return both a promise and also invoke cy commands.

Cy commands themselves are already promise like, and you can likely avoid the use of the separate Promise.

https://on.cypress.io/returning-promise-and-commands-in-test
```